File: bench/benchmarks/26_json_parse/bench.js
const doc = JSON.stringify({users: Array.from({length: 200}, (_, i) => ({
  id: i, name: `user${i}`, tags: ["alpha", "beta", "gamma"],
  scores: [1.5, 2.5, 3.5, 4.5], active: i % 2 === 0}))});
let acc = 0;
for (let i = 0; i < 100; i++) acc += JSON.parse(doc).users.length;
console.log(acc);

File: bench/benchmarks/26_json_parse/bench.rb
require 'json'
doc = JSON.generate({users: (0...200).map { |i| {id: i, name: "user#{i}",
  tags: %w[alpha beta gamma], scores: [1.5, 2.5, 3.5, 4.5], active: i.even?}}})
acc = 0
100.times { acc += JSON.parse(doc)["users"].length }
puts acc
